src/utils/gitHubUtils.ts

```diff
@@ -113,8 +111,19 @@ export async function createGitHubRequestOptions(gitHubAccessToken: string, url:
 
 export async function getGitHubAccessToken(): Promise<string> {
     const scopes: string[] = ['repo', 'workflow', 'admin:public_key'];
-    return (await vscode.authentication.getSession('github', scopes, { createIfNone: true })).accessToken;
-
+    // https://github.com/microsoft/vscode-azurestaticwebapps/issues/153
+    // tslint:disable-next-line:no-constant-condition
+    while (true) {
```

Comment:
Do we still need this since you're not getting the token on activation? Would like to avoid a loop like this if possible

Reply:
It could probably still happen, but considering that I had never seen it prior to this PR, it's probably safe to remove.